data_relay.py

```diff
@@ -8,7 +8,9 @@
 from comm_server import TelemetryFactory, ProducerToManyClient
 from telem_producer import TelemetryProducer
 
-db_type = [(
+db_type =(
+        (
+        ('h', 'PacketID'), #1
```

Comment:
This is what I think you had more inmind....Problem is that the PacketID is now being written to the CSV....which is not ideal. I could hard-code it out, however that seems a little sketchy.